tests/validation/validate_ball_and_stick.cpp

```diff
@@ -21,66 +21,75 @@
 
 using namespace nest::mc;
 
-// TODO: further consolidate common code
+struct sampler_info {
+    const char* label;
```

Comment:
You use `const char*` instead of `std::string` in a lot of places here. What is the reason for that?